### ISSUE TITLE
Error in method Feedback.getFeedback 

### ISSUE THREAD

**Issue by mrFloony** [nulab/zxcvbn4j#82]
The error is that to find the longest match, the first value in the list is taken, and then you form a sublist, but there are no values in this sublist.
Test results:

```
password: [justshort]
matches: 2
match index=0: Match{pattern=Dictionary, i=0, j=3, token=just, dictionaryName='us_tv_and_film'}
match index=1: Match{pattern=Dictionary, i=4, j=8, token=short, dictionaryName='english_wikipedia'}

matches sublist: 0

first longest=Match{pattern=Dictionary, i=0, j=3, token=just, dictionaryName='us_tv_and_film'}
last longest=Match{pattern=Dictionary, i=0, j=3, token=just, dictionaryName='us_tv_and_film'}
```

When forming sequence.subList, the toIndex parameter of the sublist method was specified incorrectly.
Test results:
```
password: [justshort]
matches: 2
match index=0: Match{pattern=Dictionary, i=0, j=3, token=just, dictionaryName='us_tv_and_film'}
match index=1: Match{pattern=Dictionary, i=4, j=8, token=short, dictionaryName='english_wikipedia'}

matches sublist: 1
match index=0: Match{pattern=Dictionary, i=4, j=8, token=short, dictionaryName='english_wikipedia'}

first longest=Match{pattern=Dictionary, i=0, j=3, token=just, dictionaryName='us_tv_and_film'}
replaced longest by match: Match{pattern=Dictionary, i=4, j=8, token=short, dictionaryName='english_wikipedia'}
last longest=Match{pattern=Dictionary, i=4, j=8, token=short, dictionaryName='english_wikipedia'}

```
Fixed in pull request #83

**Comment by vvatanabe** [comment]
@mrFloony Thanks for PR! I will check it.

**Comment by vvatanabe** [comment]
@mrFloony I released 1.3.4 include #83. Thanks!
